kafka-connect-temp/src/main/scala/com/datamountaineer/streamreactor/temp/ErrorPolicySettings.scala

```diff
@@ -4,6 +4,9 @@ import com.datamountaineer.streamreactor.connect.errors.{ErrorPolicy, ErrorPolic
 
 trait ErrorPolicySettings extends BaseSettings {
   val errorPolicyConstant: String
+  val retryIntervalConstant: String
```

Comment:
Some connectors implement ErrorPolicy but not everyone of them implements Retry Interval. So it does not make sense to make them implement both.
See for example Cassandra